src/satosa/context.py

```diff
@@ -10,16 +10,56 @@ class SATOSABadContextError(SATOSAError):
     pass
 
 
+def get_auth_req_params(context):
+    return context.state.get(Context.KEY_AUTH_REQ_PARAMS) or context.get_decoration(Context.KEY_AUTH_REQ_PARAMS) or {}
+
+
+def get_prompt_list(context):
+    auth_req_params = get_auth_req_params(context)
+    prompt = auth_req_params.get(Context.KEY_PROMPT, "")
+    return prompt if isinstance(prompt, list) else prompt.split(" ")
+
+
+def prompt_to_saml_param(context, saml_param):
+    prompt_list = get_prompt_list(context)
+    if saml_param == Context.KEY_SAML_IS_PASSIVE:
+        return "none" in prompt_list
```

Comment:
While this may seem reasonable, it is an important assumption that we cannot hardcode.

This should be a configurable choice.

Reply:
Definition of `IsPassive`:

```
If "true", the identity provider and the user agent itself MUST NOT visibly take control
of the user interface from the requester and interact with the presenter in a noticeable fashion.
```

Definition of `prompt=none`:

```
The Authorization Server MUST NOT display any authentication or consent user interface pages. An error is returned if an End-User is not already authenticated or the Client does not have pre-configured consent for the requested Claims or does not fulfill other conditions for processing the request.
```

I believe the meaning is the same, but if you want, I can introduce a configuration option.

Reply:
configuration introduced